src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/DbContextEditorServices.cs

```diff
@@ -122,18 +135,21 @@ public EditSyntaxTreeResult EditStartupForNewContext(ModelType startUp, string d
                     .FirstOrDefault(n => n is MethodDeclarationSyntax
                         && ((MethodDeclarationSyntax)n).Identifier.ToString() == "ConfigureServices") as MethodDeclarationSyntax;
 
-                if (configServicesMethod != null)
+                var configRootProperty = TryGetIConfigurationRootProperty(startUp.TypeSymbol);
+
+                if (configServicesMethod != null && configRootProperty != null)
                 {
                     var servicesParam = configServicesMethod.ParameterList.Parameters
                         .FirstOrDefault(p => p.Type.ToString() == "IServiceCollection") as ParameterSyntax;
 
                     if (servicesParam != null)
                     {
+                        AddConnectionString(dbContextTypeName, dataBaseName);
```

Comment:
it is not called if configRootProperty = null , it is different form original code flow - is that intentional?